src/main/resources/static/views/project/ProjectController.js

```diff
@@ -1,4 +1,115 @@
 var ProjectController = angular.module('greatStartApp')
-    .controller('ProjectController', ['$scope', function ($scope) {
+    .controller('ProjectController', function ($rootScope, $scope, $uibModal, $routeParams, Project, $location) {
+        var investedAmount = function (project) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "investedAmount" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of the unused 'investedAmount' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AUnusedVariable)